flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1435,6 +1584,8 @@ def test_equal(self):
         """
         Two ``DestroyBlockDeviceDataset`` instances compare as equal if they
         are initialized with the same volume.
+
+        XXX: Move to shared mixin.
```

Comment:
Maybe not worth doing a lot of refactoring for these composed IStateChange implementations yet.  We're going to twiddle the implementation around a bit more before we find the final shape for them, I think.  We want something that's both easier to define and more introspectable than what ended up in the FLOC-1582 branch (which I expect will be much the same as what ends up in this branch).